tests/test_uart.py

```diff
@@ -1,25 +1,24 @@
-import xmostest
 
+import sys
+import pytest
 
-class UartTester(xmostest.Tester):
+class UartTester():
     def __init__(self):
-        super(UartTester, self).__init__()
-        self.register_test("lib_uart", "sim_regression", "loopback_test", {})
+        self.str = "hello"
```

Comment:
Required?

Reply:
not really, I wasn't sure you can make a class without the __init__